examples/datasource-http-backend/src/plugin.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/grafana/grafana/main/docs/sources/developers/plugins/plugin.schema.json",
   "type": "datasource",
   "name": "Datasource Http Backend",
-  "id": "example-httpbackend-datasource",
+  "id": "myorg-httpbackend-datasource",
```

Comment:
sorry, this breaks some of my devs dashboards and other stuff, can we keep the same plugin ID? (I know it's incosistent between examples though).

Reply:
We can sure but I was thinking it might be a good idea to make the IDs consistent at some point. Considering this is dev only, I figured breaking changes would be acceptable 🤷

Reply:
yeah, not a big deal but it'd be still incosistent, some of them use `grafana` as the org, others `myorg` and others `example` 🤷

Reply:
No that's what I meant 🙈 https://github.com/grafana/grafana-plugin-examples/pull/485